docker/legacy-auth/compose.yaml

```diff
@@ -2,6 +2,10 @@
 # For Proxy Server instructions please see
 #  https://github.com/voxel51/fiftyone-teams-app-deploy/tree/main/docker#environment-proxies
 services:
+  fiftyone-app:
```

Comment:
Moved the `fiftyone-app` service to match the layout in `docker/internal-auth` to make diffs easier.